pubpub/pubpub
# feature: Collection layouts at /:slug

Related to #1055 

This PR allows you to visit a Collection at `/:slug` or `/collection/:slug`, where we render its Layout or redirect you to a linked Page as appropriate. We continue to support passing in the first eight digits of the collection UUID in place of a slug, to support URLs using that scheme that have been deposited to Crossref.

_Test plan:_
- From a Collection dashboard, click "Visit Collection" and verify that this link (containing the Collection slug) takes you to the Collection's layout. For a Collection with a linked Page, it should bring you to the Page instead.
- Verify that Pages are also still available at `/:slug` (we'll make sure their slugs can't collide later on)
- Confirm that you can now add Collections to the Community navigation (!!)
- Check that `/collection/:uuid.slice(0,8)` still resolves to a Collection as expected.
- Check that the `collectionUrl` utility works as expected, e.g. by clicking on the Collections in a Pub header.
